build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -775,20 +1245,43 @@ new file mode 100644
 +#include "base/strings/safe_sprintf.h"
 +#include "base/strings/string_number_conversions.h"
 +#include "base/strings/string_split.h"
++#include "components/prefs/pref_service.h"
++#include "components/prefs/pref_registry_simple.h"
++#include "chrome/common/pref_names.h"
++
 +namespace adblock_updater {
 +
++// Holds the URL to an indexed subresource filters file.
++const char kAdBlockFiltersURL[] = "adblock.filters_url";
++
++// Holds the URL to an indexed subresource filters file.
++const char kAdBlockFiltersCheckPeriod[] = "adblock.check_period";
++
++// Last check time
++const char kAdBlockLastCheckTime[] = "adblock.last_check_time";
++
++// Last check time (only with ok)
++const char kAdBlockLastCheckTimeOk[] = "adblock.last_check_time_ok";
++
 +// all constants express seconds
 +// these could be made configurable
 +const int initial_check_delay = 5,
-+      next_check_delay = 60*60*24*7, // 1 week
 +      on_demand_check_delay = 60; // minimum 1 minute between each on-demand check
-+
-+AdBlockUpdaterService::AdBlockUpdaterService(scoped_refptr<network::SharedURLLoaderFactory> shared_url_network_factory, std::unique_ptr<component_updater::UpdateScheduler> scheduler,
-+   subresource_filter::RulesetService* ruleset_service, std::string filters_url)
-+ : ruleset_service_(ruleset_service), shared_url_network_factory_(shared_url_network_factory), scheduler_(std::move(scheduler)) {
++int next_check_delay = 60*60*24*7; // 1 week
```

Comment:
If this is not a constant then it should be on the `AdBlockUpdaterService` class (`next_check_delay_`)

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/b11ab2588e787d6349664120c0ccdb7827cab201